course/src/main/java/in/testpress/course/util/ExoPlayerUtil.java

```diff
@@ -369,30 +365,6 @@ public void onAnimationEnd() {
                 });
     }
 
-
-    private void initializeAudioManager() {
-        audioManager = (AudioManager) activity.getSystemService(AUDIO_SERVICE);
-        initAudioFocusChangeListener();
-        audioManager.requestAudioFocus(audioFocusChangeListener, AudioManager.STREAM_MUSIC, AudioManager.AUDIOFOCUS_GAIN);
-    }
-
-    private void initAudioFocusChangeListener() {
-        audioFocusChangeListener = new AudioManager.OnAudioFocusChangeListener() {
-            public void onAudioFocusChange(int focusChange) {
-                switch (focusChange) {
-                    case (AudioManager.AUDIOFOCUS_LOSS_TRANSIENT):
-                        player.pause();
-                        break;
-                    case (AudioManager.AUDIOFOCUS_LOSS) :
-                        player.pause();
-                        break;
-                    default:
-                        break;
-                }
-            }
-        };
-    }
```

Comment:
Where is alternative code for this

Reply:
Added in this - https://github.com/testpress/android-sdk/pull/447/files#diff-e97710cc9db1366e5d27d271a027b3ff25766e61daf1cdd1595b2475369b1675R319